### PR TITLE
maint: remove drippie-mon

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Drippie monitoring is now handled in monitorism.

### CODE REVIEW

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
Needs a review from @ethereum-optimism/security-reviewers

**Comment by zchn** [comment]
can we get an LGTM from @ajsutton to ensure knowledge & context is maintained within proofs team? I'll approve after @ajsutton approves since we recently agreed that drippie-mon will be owned by proofs team, not security team.

**Review by ajsutton — APPROVED** [review]
Yes Drippie mon has been moved into monitorism.

**Review by zchn — APPROVED** [review]

**Pull request merged**
